src/utils.jl

```diff
@@ -1,6 +1,8 @@
 # ------------------------------------------------------------------
 # Licensed under the MIT License. See LICENSE in the project root.
 # ------------------------------------------------------------------
+using JSON3
+using CoordRefSystems
```

Comment:
We do not import modules in source files. The main GeoIO.jl file already imports these.